nwcsafpps_runner/prepare_nwp.py

```diff
@@ -245,18 +238,35 @@ def check_nwp_content(gribfile):
         LOG.warning("Cannot check if NWP files is ok!")
         return True
 
-    srplines = [ll.strip('M ').strip('\n')
-                for ll in lines if str(ll).startswith('M')]
-
+    srplines = [ll.strip('M ').strip('\n') for ll in lines if str(ll).startswith('M')]
```

Comment:
These lines here are a bit cryptic. Wonder if it would be more clear if they were put in separate functions with meaningful names?